validation/serializers.py

```diff
@@ -13,16 +13,18 @@ def validate_edrpou(value):
         return value
     
     @staticmethod
-    def validation_email(email_to_validate: str):
+    def validate_contact_email(email_to_validate: str):
         pattern = r'([A-Za-z0-9]+[.-_])*[A-Za-z0-9]+@[A-Za-z0-9-]+(\.[A-Z|a-z]{2,})+'
         if not re.match(pattern, email_to_validate):
             raise ValidationError("Email is invalid")
+        return email_to_validate
```

Comment:
Do we use this method in the project?
We have to rename it everywhere, check please.

Reply:
This method is automatically invoked by the serializer; DRF does this using the pattern validate_<fieldname>.